packages/atomic/src/components/commerce/facets/atomic-commerce-facet/e2e/atomic-commerce-facet.e2e.ts

```diff
@@ -14,8 +15,8 @@ test.describe('default', () => {
   test('should allow to filter by selecting and deselecting a value', async ({
     facet,
   }) => {
-    const facetValueLabel = facet.getFacetValue('Nike');
-    const facetValueBtn = facet.getFacetValueButton('Nike');
+    const facetValueLabel = facet.getFacetValue('Ecco');
+    const facetValueBtn = facet.getFacetValueButton('Ecco');
```

Comment:
the tests failed because of changes in the merch hub ?

Reply:
New products pushed to barca